payloads/api_particulier_v2_cnav_quotient_familial_v2/fake_france_connect_cnaf.yml

```diff
@@ -5,13 +5,13 @@ description: |-
   L'endpoint est appellé avec le jeton FranceConnect + le recipient.
 
 params:
-  given_name: "Georges"
-  family_name: "CNAF"
-  birthdate: "2002-01-01"
-  gender: "male"
-  birthplace: "75002"
-  birthcountry: "99100"
-  preferred_username: "MARTIN"
+  codeInseeLieuDeNaissance: "75002"
+  codePaysLieuDeNaissance: "99100"
+  nomNaissance: "CNAF"
+  nomUsage: "MARTIN"
+  prenoms: ["Georges"]
+  recipient: "13002526500013"
```

Comment:
Faudrait supprimer cette clé et l'ignorer dans siade dans la méthode `mocking_params` (si ce n'est pas déjà fait).

Reply:
Bien vu, je vais creuser ça (sûrement la semaine pro).

D'ailleurs j'ai noté quelques autres trucs "faciles" sur `MockDataBackend `j'ouvrirai une PR pour qu'on en discute :ok:

Reply:
Mon point est que `recipient` ne devrait pas servir dans le cadre du fetch du mock pour QF v2 : quelque soit le recipient que l'on met on veut que pour des params d'identité cela fonctionne. Cette remarque est aussi valide pour le `request_id` qu'on ne peut de toute manière pas déterminé en amont.

Ce qui est passé au service de mock est `mocking_params`. Si on remonte le fil:

1. Le `mocking_params` qui nous intéresse: https://github.com/etalab/siade/blob/f2a1458ee4f6657816ccdcf0b2ff7a05d78e9db5/app/interactors/cnav/quotient_familial_v2/make_request.rb#L5
2. .. qui remonte ici https://github.com/etalab/siade/blob/f2a1458ee4f6657816ccdcf0b2ff7a05d78e9db5/app/interactors/cnav/make_request.rb#L8
3. .. et pour FC -> https://github.com/etalab/siade/blob/f2a1458ee4f6657816ccdcf0b2ff7a05d78e9db5/app/interactors/cnav/make_request.rb#L62

J'en déduis que le `recipient` est bien ignoré, et qu'à priori il suffit ici de double-check que les mocking params sont bien définis (ce qui me semble OK ici)